java/src/main/java/dojo/supermarket/model/ShoppingCart.java

```diff
@@ -48,7 +48,10 @@ void handleOffers(Receipt receipt, Map<Product, Offer> offers, SupermarketCatalo
                 } else if (offer.offerType == SpecialOfferType.TwoForAmount) {
                     x = 2;
                     if (quantityAsInt >= 2) {
-                        double total = offer.argument * (quantityAsInt / x) + quantityAsInt % 2 * unitPrice;
```

Comment:
hmm I think some java changes have sneaked into this pull request, can you remove this one?